contrib/TestHarness2/test_harness/run.py

```diff
@@ -195,6 +196,8 @@ def parse_txt(self, path: Path):
                 test_class = test_name
             if priority is None:
                 priority = 1.0
+            if is_rare(path) and priority <= 1.0:
```

Comment:
Curious, what's the reason behind having `priority <= 1.0` in this condition? 

PR descriptions says: "to override the default 10 when testPriority is not set in the test TOML file.". Is that the reason? If yes, then why not have env variable override even if testPriority is set?

Reply:
If `testPriority` is not set in the toml file, then 1.0 is used. In this case, I want to override for the rare tests. However, some of the tests have been set to other priorities in the toml file, in which case I don't want to override the values.

Reply:
Makes sense